### ISSUE TITLE
LineChart: Reverse focus order does not work with Keyboard navigation from Legends to Chart

### ISSUE THREAD

**Issue by RishabhPokharna** [microsoft/fluentui#18833]
### Environment Information

- **Package version(s)**: @uifabric/charting (Version "4.15.32")
- **Browser and OS versions**: Edge (Chromium), Windows 10

### Describe the issue:
In the LineChart component from @uifabric/charting npm package (Version "4.15.32"), while using Keyboard navigation, Reverse focus order (Using Shift+Tab key) does not work from the legends to the chart lines (except the first legend)

### Please provide a reproduction of the issue in a codepen:
Issue is reproducible in the examples provided here. The example I used is "**Line Chart Basic**".
https://admincontrolsdemoapps.blob.core.windows.net/release/charting-demo/3.0.1/index.html#/examples/LineCharts
**Steps to reproduce:**
1. Using Keyboard navigation, move the focus to "All" legend.
2. Press SHIFT+Tab key. The focus moves to `</> Show Code` component. Expected behavior is it should move to the first point in the chart.

#### Actual behavior:
While using keyboard navigation, when focus is currently set to one of the legends (except first one), pressing SHIFT+Tab key moves the focus to the component before the Chart.

#### Expected behavior:
While using keyboard navigation, when focus is on any of the legends, pressing the SHIFT+Tab key moves the focus to the chart.

### Documentation describing expected behavior
[MAS 2.4.3 - Focus Order](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9%7d)

**Comment by ling1726** [comment]
Assining @Raghurk  since they own the charting package now.

**Comment by gouttierre** [comment]
@RishabhPokharna - thanks for reaching out to us.

@Raghurk - Would you be able to confirm if this is a regression, or if this behavior is an issue?

**Comment by RishabhPokharna** [comment]
+cc: @Ramesh ***@***.***>, @Vibhor ***@***.***>, @Anupam ***@***.***>
________________________________
